packages/utils/arguments_parser.ts

```diff
@@ -0,0 +1,39 @@
+import * as t from "@babel/types";
+
+export type Dependency = number | string | { uniqueId: string; }
+
+export type Options = {
+  webpackChunkName?: string;
+}
+
+function isDependency(element: any): element is Dependency {
+  return typeof element === 'number' || typeof element === 'string' || (typeof element === 'object' && element.uniqueId !== undefined);
+}
+
+export function parseSSRComputationArgs(parent: t.CallExpression): { filename: string, dependencies: Dependency[] } {
```

Comment:
How is this function used? I can't find a call :thinking:

Reply:
Sorry, was initial thoughts. I deleted it.